docs/Java_API.md

```diff
@@ -1,10 +1,35 @@
 # ONNX Runtime Java API
 The ONNX runtime provides a Java binding for running inference on ONNX models on a JVM, using Java 8 or newer.
 
-Two jar files are created during the build process, one contains the onnxruntime shared library, the JNI binding and the Java class files, and the other only contains the class files. By default the shared libraries are loaded from the classpath in a folder called `/lib`, if you wish to have them load from `java.library.path` then supply `-DORT_LOAD_FROM_LIBRARY_PATH` to the JVM at runtime.
+Release artifacts are published to Maven Central for use as a dependency in most Java build tools:
+
+* [com.microsoft.onnxruntime:onnxruntime](https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime) - cross-platform CPU artifact
+* [com.microsoft.onnxruntime:onnxruntime_gpu](https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime_gpu) - cross-platform GPU (CUDA) artifact
+
+These artifacts are currently built for common OS/Architecture combinations: Windows x64, Linux x64, MacOS x64.
+
+For building locally, please see the [Java API development documentation](../java/README.md) for more details.
+
+For customization of the loading mechanism of the shared library, please see [advanced loading instructions](../java/README.md#advanced-loading).
+
+## API Reference
+
+The Javadoc is available [here](https://microsoft.github.io/onnxruntime/java/index.html).
```

Comment:
@faxu we should update with the Javadoc for v1.3.1, as this is still the older Javadoc.

Reply:
I found this https://stackoverflow.com/a/58465995

We publish the javadoc to maven alongside our artifacts, so this site basically takes the javadoc from there and displays it in a CDN-backed webpage for free. This way we do not have to keep upgrading the gh-pages branch. Maybe the old /java/index.html will need to redirect for posterity.

https://javadoc.io/doc/com.microsoft.onnxruntime/onnxruntime

It also handles different versions which is nice.

Is this a reasonable solution?

Alternatively, could the javadoc be built and pushed to the gh-pages branch using the azure CI process?

Reply:
I agree that seems reasonable, but it's up to MS if they want to host the docs themselves or push it off to some third party.

I think building the javadoc as part of the gh-pages is reasonable, I don't recall how they got built initially.